contracts/Aavegotchi/libraries/LibAavegotchi.sol

```diff
@@ -159,7 +159,7 @@ library LibAavegotchi {
             aavegotchiInfo_.equippedWearables = s.aavegotchis[_tokenId].equippedWearables;
             aavegotchiInfo_.collateral = s.aavegotchis[_tokenId].collateralType;
             aavegotchiInfo_.escrow = s.aavegotchis[_tokenId].escrow;
-            aavegotchiInfo_.stakedAmount = IERC20(aavegotchiInfo_.collateral).balanceOf(aavegotchiInfo_.escrow);
+//            aavegotchiInfo_.stakedAmount = IERC20(aavegotchiInfo_.collateral).balanceOf(aavegotchiInfo_.escrow);
```

Comment:
I think we will tell users to pull out their collateral 
two choices:
1. pull out all collateral
2. allow users to stake erc20 GHST instead

so when a gotchi bridges, we should require(stakedAmount < 0.001)

Reply:
maybe could just unstake it for them during bridging